src/services/weather.js

```diff
@@ -30,6 +34,28 @@ async function fetchWeatherData(city) {
   return mapModelToView(response.data);
 }
 
-function getIconURL(icon) {
+export function getIconURL(icon) {
   return weatherImageURL + icon + imageExtension;
 }
+
+function mapModelForecastToView(model) {
```

Comment:
I don't think this function is necessary as it's only a wrapper for model that is a list.
I would just use `mapModelToView` instead.